### DISCUSSION TITLE
Some files not generated

### DISCUSSION THREAD

**Discussion by caphector** [Daveiano/weewx-wdc#190] [Q&A]
I installed WDC skin v3.3.0 on my weewx v4.8.0. The install completed normally and it added the `[[WdcReport]]` block to my conf file. I set the `HTML_ROOT` inside `[[WdcReport]]` to a new empty folder in my weewx folder. I restarted weewx and ran `wee_reports`.

When `wee_reports` completed I checked the target folder and I'm missing some files - there's no `index.html`, for example. I've tried changing the config (commented out most of my config options, trying different folders with `HTML_ROOT`, different values for `base_path`) but so far nothing has made a difference. The only odd thing in the logs is that 20 files are generated and 19 copied after I empty out `HTML_ROOT`:

```
wee_reports[8625] INFO weewx.cheetahgenerator: Generated 20 files for report WdcReport in 46.12 seconds
wee_reports[8625] INFO weewx.reportengine: Copied 19 files to /home/weewx/wdc/
```

Edit: Reading comprehension fail. I was on weewx v4.8.0; v4.9 or later is required. Updated and it worked as expected.

**Comment by Daveiano** [comment]
Thanks for reaching out! Yes, the skin requires at least WeeWX 4.9.

Glad you could solve this!
